docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -267,385 +280,424 @@
 			<td><img alt="" height="267" src="media/m1appcommon.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/AppCommon" target="_blank"><strong>App-common</strong></a></p>
-			<p>The sample application demonstrates how to work with app-common API in order to obtain application-specific information.</p>
+			<p>This sample application demonstrates how to work with app-common API in order to obtain application-specific information.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m2applicationcontrol.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/ApplicationControl" target="_blank"><strong>Application Control</strong></a></p>
-			<p>This Application Control sample application demonstrates how you can call the operation of other applications that provide operations such as pick, view, or compose.</p>
+			<p>This sample application demonstrates how you can call the operation of other applications that provide operations such as pick, view, or compose.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m3volume.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/VolumeController" target="_blank"><strong>Volume</strong></a></p>
-			<p>Volume application demonstrates how to retrieve and set device's volume levels.</p>
+			<p>This sample application demonstrates how to retrieve and set device's volume levels.</p>
```

Comment:
This sample application demonstrates how to retrieve and set the device volume levels.

Reply:
fixed